lib/usage-tracking/tests/support/class-usage-tracking-test-subclass.php

```diff
@@ -37,4 +37,31 @@ public function current_user_can_manage_tracking() {
 	public function opt_in_dialog_text() {
 		return 'Please enable Usage Tracking!';
 	}
+
+	public function do_track_plugin( $plugin_slug ) {
+		if ( in_array( $plugin_slug, array( 'hello', 'test', 'my-favorite-plugin' ) ) ) {
```

Comment:
Linter: `Not using strict comparison for in_array; supply true for third argument.`